2/README.md

```diff
@@ -0,0 +1,122 @@
+
+### Retired Specifications
+When deprecated specifications are no longer used in products, they become **retired** specifications. Retired specifications are part of the historical record. They should not be changed except to indicate their replacements, if any. Retired specifications have no contractual weight.
+
+### Deleted Specifications
+Deleted specifications are those that have not reached maturity (stable) and were discarded. They should not be used and are only kept for their historical
+value. Only Raw and Draft specifications can be deleted.
+
+## Editorial control
+A specification MUST have a single responsible editor, the only person
+who SHALL change the status of the specification through the lifecycle stages.
+
+A specification MAY also have additional contributors who contribute changes to it. It is RECOMMENDED to use the [C4 process](../1/README.md) to maximize the scale and diversity of contributions.
+
+The editor is responsible for accurately maintaining the state of specifications and for handling all comments on the specification.
+
+## Branching and Merging
+Any member of the domain MAY branch a specification at any point. This is done by copying the existing text, and creating a new specification with the same name and content, but a new number. The ability to branch a specification is necessary in these circumstances:
+
+* To change the responsible editor for a specification, with or without the cooperation of the current responsible editor.
+* To rejuvenate a specification that is stable but needs functional changes. This is the proper way to make a new version of a specification that is in stable or deprecated status.
+* To resolve disputes between different technical opinions.
+
+The responsible editor of a branched specification is the person who makes the branch.
+
+Branches, including added contributions, are derived works and thus licensed under the same terms as the original specification. This means that contributors are guaranteed the right to merge changes made in branches back into their original specifications.
+
+Technically speaking, a branch is a *different* specification, even if it carries the same name. Branches have no special status except that accorded by the community.
+
+## Conflict resolution
+COSS resolves natural conflicts between teams and vendors by allowing anyone to define a new specification. There is no editorial control process except that practised by the editor of a new specification. The administrators of a domain (moderators) may choose to interfere in editorial conflicts, and may suspend or ban individuals for behaviour they consider inappropriate.
+
+## Conventions
+Where possible editors and contributors are encouraged to:
+
+* Refer to and build on existing work when possible, especially IETF specifications.
+* Contribute to existing specifications rather than reinvent their own.
+* Use collaborative branching and merging as a tool for experimentation.
+
+## License
```

Comment:
The GNU General Public License is for software and is completely inappropriate for something like specs, but I guess they used it because it was available at the time. Today, there are many more appropriate licenses, such as the GNU Free Documentation License (GFDL) and the various CC licenses.

As mentioned above, we could just ask contributors to dedicate their contributions (specs) to the public domain using CC0, i.e. https://creativecommons.org/publicdomain/zero/1.0/

Reply:
I'd love to have C4 and COSS under CC0, but the original work has been licensed using GPL, so we need to release our changes under the same license.